examples/basic/BasicSendWithProxy/BasicSendWithProxy.go

```diff
@@ -3,15 +3,16 @@ package main
 import (
 	"fmt"
 
-	"github.com/socketlabs/socketlabs-go/examples"
-	"github.com/socketlabs/socketlabs-go/injectionapi"
-	"github.com/socketlabs/socketlabs-go/injectionapi/message"
+	"github.com/PraneethChandraThota/socketlabs-go/examples"
```

Comment:
after your tests are complete please return the imports back to the main repo

Reply:
Changed back to the main repo